cmd/podman/login.go

```diff
@@ -90,6 +91,19 @@ func loginCmd(c *cliconfig.LoginValues) error {
 	}
 
 	ctx := getContext()
+
+	var password string
+	var stdinPasswordStrBuilder strings.Builder
+	if c.Flag("password-stdin").Changed {
+		if c.Username == "" {
+			return errors.Errorf("Must provide --username with --password-stdin")
+		}
+		scanner := bufio.NewScanner(os.Stdin)
+		for scanner.Scan() {
+			fmt.Fprint(&stdinPasswordStrBuilder, scanner.Text())
+		}
+		password = stdinPasswordStrBuilder.String()
+	}
```

Comment:
is there any cli validation that should occur between this flag and the password flag? exclusive?

Reply:
Please explain, what you re worried about?

Reply:
We have `--password` and `--password-stdin` - they should be mutually exclusive.

Reply:
They are now.